packages/core/src/lib/features/AbstractFeature.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Feature} from './Feature';
+
+export class AbstractFeature implements Feature {
```

Comment:
I'm not quite sure I'd call this AbstractFeature, to me "abstract" kinda means "somewhat implemented, but left some gaps for subclasses to implement". Following examples from the Chrome codebase, I'd probably call this "EmptyFeature". It reads quite nicely too, "AppsFlyerFeature extends EmptyFeature but with these bits filled in".

Reply:
SGTM - naming is hard :P

Reply:
Renamed.